genesis/src/main.rs

```diff
@@ -30,17 +34,26 @@ pub enum AccountFileFormat {
     Keypair,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct PrimordialAccountDetails {
```

Comment:
accounts are serialize-able.  any reason not to use them directly in the file?

Reply:
@rob-solana considered that, reason is since genesis is going to be prepared and edited by human, we wanted to use human-friendly serialization formats for  `Vec<u8>` and `PubKey`.

Reply:
The yaml serialization of a Pubkey and Vec<u8> is pretty ghastly.  I suspect the primordial accounts file would ultimately be created by a program but then inspected by human eyes.